app/assets/stylesheets/content/_notifications.sass

```diff
@@ -28,33 +28,95 @@
 
 // NOTE: these are demo ski
 
+//nm - notification messages
+$nm-font-size: rem-calc(13px)
+$nm-border-radius: rem-calc(2px)
+$nm-box-padding: rem-calc(10px 35px 10px 35px)
+
+$nm-color-error-border: #ca3f3f
+$nm-color-error-icon: #ca3f3f
+$nm-color-error-background: #fedada
+
+$nm-color-success-border: #35c53f
+$nm-color-success-icon: #35c53f
+$nm-color-success-background: #d8fdd1
+
+$nm-color-warning-border: #b5b078
+$nm-color-warning-icon: #000
+$nm-color-warning-background: #f4f4aa
+
+%nm-icon-error
+  @extend .icon-attention2:before
+  color: $nm-color-error-icon
+
+%nm-icon-success
+  @extend .icon-yes:before
+  color: $nm-color-success-icon
+
+%nm-icon-warning
+  @extend .icon-attention1:before
+  color: $nm-color-warning-icon
+
+
+@mixin messages-icon
+  @include icon-common
+  position: absolute
+  top: rem-calc(11px)
+  left: rem-calc(10px)
+  font-size: rem-calc(18px)
+
 .notification-box
```

Comment:
I agree. There could be variables for this in the same namespace (`$notification-box--`). I totally fogot for a moment there, that the blue, yellow and green might collide in the MyProject theme - and they might not be accessible, so easy changing should be possible.